about.html

```diff
@@ -0,0 +1,95 @@
+    </div>
+  </section>
+  <section id="about-info">
+    <div class="wrapper">
+      <h2>Post High-School Story</h2>
+      <hr>
+      <p>At university. we found out that they gave scholarships for being part of their cultural groups. They had groups of dance, choir, painting, cinema and among those groups we found the ENSEMBLE DE CUERDAS that was directed by Iván Jumbo. That club had the premise of doing instrumental covers, with classical guitars only, of "Pasillo", Samba, Andean music, classical music and many other different genres of music. This extra-academic activity opened up our musical horizons and had a great influence on our current compositions.</p>
+      <p>At the same time, together with some friends, we created our first group named Doppler, with which we did several gigs and two original songs. We even took part in a music festival in Quito.</p>
+      <p>Now, we are developing new music combining this influences and learning every day. We are very excited to present our first album as Dasik, "Árbol Blanco".</p>
+      <p>Please contact us per email for any further questions about CC Global Summit 2015!</p>
+      <p><a href="mailto:dasikband@gmail.com" class="about-email">dasikband@gmail.com</a></p>
+    </div>
+  </section>
+  <section id="past-events">
+    <div class="wrapper">
+      <h2>See Past Gigs</h2>
+      <hr>
+      <p>Check previous events and concerts.</p>
+      <ul id="events">
+        <li>
+          <img src="./images/LaAldeaGig.jpeg" alt="La Aldea">
+          <div class="overlay">
+            <h3>2022</h3>
+            <p>"La Aldea" Pub Gig</p>
+          </div>
+        </li>
+        <li>
+          <img src="./images/AfterGig2.jpeg" alt="After">
+          <div class="overlay">
+            <h3>2021</h3>
+            <p>"After" Pub Gig</p>
+          </div>
+        </li>
+      </ul>
+    </div>
+  </section>
+  <footer id="copyright" class="cp-dark">
+    <div class="wrapper">
+      <div class="footer-logo">
+        <img id="img-white" src="./images/cc-logo-white.png" alt="Creative Commons">
+        <img id="img-dark" src="./images/cc-logo-dark.png" alt="Creative Commons">
+      </div>
+      <div class="cp-info">
+        <p><a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/"><img alt="Creative Commons License" src="https://i.creativecommons.org/l/by-nc/4.0/88x31.png" /></a></p>
+        <p>2022 Creative Commons. Some Rights Reserved</p>
+        <p>This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">Creative Commons Attribution-NonCommercial 4.0 International License</a>.</p>
+      </div>
+    </div>
+  </footer>
```

Comment:
- [ ] [OPTIONAL] Kindly remove the error code displayed below your footer, see the image below
![image](https://user-images.githubusercontent.com/61408860/169390351-42a3ebbb-6ce3-47ae-b47e-f7cd8dcb0b2c.png)